src/main/java/com/direwolf20/buildinggadgets/items/CopyPasteTool.java

```diff
@@ -128,66 +122,11 @@ public static Integer getLastBuildDim(ItemStack stack) {
         return GadgetUtils.getDIMFromNBT(stack, "lastBuild");
     }
 
-    public static void setItemCountMap(ItemStack stack, Map<UniqueItem, Integer> tagMap) {
-        NBTTagCompound tagCompound = stack.getTagCompound();
-        if (tagCompound == null) {
-            tagCompound = new NBTTagCompound();
-        }
-        NBTTagList tagList = GadgetUtils.itemCountToNBT(tagMap);
-        tagCompound.setTag("itemcountmap", tagList);
-        stack.setTagCompound(tagCompound);
-    }
-
-    public static Map<UniqueItem, Integer> getItemCountMap(ItemStack stack) {
-        NBTTagCompound tagCompound = stack.getTagCompound();
-        if (tagCompound == null) {
-            return null;
-        }
-
-        Map<UniqueItem, Integer> tagMap = GadgetUtils.nbtToItemCount((NBTTagList) tagCompound.getTag("itemcountmap"));
-        return tagMap;
-    }
-
-    public static void setStartPos(ItemStack stack, BlockPos startPos) {
-        GadgetUtils.writePOSToNBT(stack, startPos, "startPos");
+    public static ArrayList<BlockMap> getBlockMapList(@Nullable NBTTagCompound tagCompound) {
```

Comment:
Could we move to the Interface here?

Reply:
I only moved the methods that were duplicated in both classes to the interface. This method seems to only be used in the context of the copy/paste tool.